### PR TITLE
arrowpath PR requested by <@647041687541121044>

### PR DESCRIPTION
**Pull request by RayDeeUx**
the actual jar file comes next hold on tight

### CODE REVIEW

**Review by nacrt — CHANGES_REQUESTED** [review]
remove the url

**Comment by RayDeeUx** [comment]
wait @nacrt which one the showcase or the jar file url

**Comment by nacrt** [comment]
"url": "..."

File: files/mods.json [hunk 1]
```diff
@@ -1980,6 +1980,27 @@
...
+        "actions": [
+            {
+                "icon": "youtube.png",
+                "text": "Showcase",
+                "link": "https://www.youtube.com/watch?v=f3iP-lzobxk"
+            }
+        ],
+        "categories": [
+            "5;All PvP",
+            "6;Recommended PvP"
```
Comment (KTibow) [inline files/mods.json]: sus

Comment (nacrt) [reply files/mods.json]: this should also be removed from recommend

**Pull request merged**